sources/McQuery.Net/Exceptions/AlreadyExpiredException.cs

```diff
@@ -0,0 +1,17 @@
+using McQuery.Net.Internal.Abstract;
+
+namespace McQuery.Net.Exceptions;
+
+[PublicAPI]
+public class AlreadyExpiredException : ArgumentException
```

Comment:
I think it can be internal. Should add public common McQuery exception instead to provide an appropriate message.
That's all we need to mark that something goes wrong I belive